contracts/RelayHub.sol

```diff
@@ -458,39 +502,12 @@ contract RelayHub is IRelayHub {
         }
     }
 
-    function requiredGas(uint256 relayedCallStipend) public view returns (uint256) {
-        return
-        GAS_OVERHEAD +
-        GAS_RESERVE +
-        ACCEPT_RELAYED_CALL_MAX_GAS +
-        PRE_RELAYED_CALL_MAX_GAS +
-        POST_RELAYED_CALL_MAX_GAS +
-        relayedCallStipend;
-    }
-
-    function maxPossibleCharge(
-        uint256 relayedCallStipend,
-        uint256 gasPrice,
-        uint256 transactionFee)
-    public
-    view
-    returns (uint256) {
-        return calculateCharge(requiredGas(relayedCallStipend), gasPrice, transactionFee);
-    }
-
-    function calculateCharge(uint256 gas, uint256 gasPrice, uint256 fee) private pure returns (uint256) {
+    function calculateCharge(uint256 gas, uint256 gasPrice, uint256 fee) public view returns (uint256) {
```

Comment:
why is it view and not pure?

Reply:
It is allowed for a RelayHub (in the future) to use some storage variable to affect the charge. Wanted to stress that out.